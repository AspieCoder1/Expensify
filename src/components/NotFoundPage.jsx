import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
	<div>
		<h1>Error 404: File Not Found</h1>
		<button><Link to="/">Go Home</Link></button>
	</div>
);

export default NotFoundPage;