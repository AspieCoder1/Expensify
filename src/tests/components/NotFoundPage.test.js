import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage.jsx';

test('should render NotFoundPageCorrectly', () => {
	const wrapper = shallow(<NotFoundPage />);
	expect(wrapper).toMatchSnapshot();
});