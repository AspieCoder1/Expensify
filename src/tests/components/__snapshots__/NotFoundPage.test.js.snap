// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPageCorrectly 1`] = `
<div>
  <h1>
    Error 404: File Not Found
  </h1>
  <button>
    <Link
      replace={false}
      to="/"
    >
      Go Home
    </Link>
  </button>
</div>
`;
